src/types.ts

```diff
@@ -55,3 +58,13 @@ export interface IServiceParams {
   project?: string
   server_uri?: string
 }
+
+export interface IRequestHeaders {
+  url: string;
+  options: {
+    method: string,
+    agent?: Agent
+    body?: string | Buffer,
+    headers: Record<string, string>
+  } 
+}
```

Comment:
Can this be replaced with an export from the types package?

Reply:
No, it's a custom object.